src/modules/launcher/PowerLauncher/MainWindow.xaml.cs

```diff
@@ -283,6 +284,13 @@ private void SuggestionsList_Tapped(object sender, TappedRoutedEventArgs e)
             }
         }
 
+        private void SuggestionList_UpdateListSize(object sender, ContainerContentChangingEventArgs e)

```

Comment:
Can we comment why, we have to hack around the Panel resizing itself?  I'm not sure I fully understand and it would be good context for someone reading this. Particularly if its a known issue with xaml islands that may be fixed.

Reply:
Sure I shall add why we had to choose this path. TBH I'm not sure of the exact reason as to why it wasn't working. Binding Height/MaxHeight Properties does not work either.